### PR TITLE
Added the Jogamp dependency and implemented FocusOMeterGLRenderer

### LINKED ISSUE #174: Neuro-Visual Feedback Module should be implemented
**Actual Behaviour**
* Currently the application lacks the implementation of the Neuro-Visual Feedback Module which includes the classes - FocusOMeter, FocusOMeterGLRenderer, LongtermFFTVisualizer, NeuroGameRenderer, NeuroGameVisuals, OSCFeedback, ZenSpaceGLRenderer, ZenSpaceVisuals.

**Expected Behaviour**
* All these classes should be correctly ported after converting them to Android Version.
* The usages of Jogamp should be appropriately substituted everywhere with the correct replacement for Android Optimization.

**Classes to Port and Checklist of the work** 
- [x] FocusOMeter
- [x] FocusOMeterGLRenderer
- [x] LongtermFFTVisualizer
- [x] NeuroGameRenderer
- [x] NeuroGameVisuals
- [x] OSCFeedback
- [x] ZenSpaceGLRenderer
- [x] ZenSpaceVisuals

**Would you like to work on the issue?**
I would like to work on this issue.

### PR DESCRIPTION
**Pull request by suresh-gandhi**
Fixes #174 (Targeting the checklist point 2)

**Changes**: 
* Added the Jogamp dependency and implemented FocusOMeterGLRenderer class.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications are done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: [Compress the `app-debug.apk` file into a `<feature>.rar` or `<feature>.zip` file and upload it here]

### CODE REVIEW

**Comment by suresh-gandhi** [comment]
@iamareebjamal Again the same issue. Looking into it. Meanwhile, if you figure out something please let me know. I guess we could even use Open GL ES 2.0 for the same purpose if this doesn't work.

File: app/src/main/java/io/neurolab/output/visual/FocusOMeterGLRenderer.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+import java.util.Random;
+import com.jogamp.opengl.GL;
+import com.jogamp.opengl.GL2;
+import com.jogamp.opengl.glu.GLU;
+
+
+public class FocusOMeterGLRenderer {
+
+    static float currentFeedback = 0;
```
Comment (adityastic) [inline app/src/main/java/io/neurolab/output/visual/FocusOMeterGLRenderer.java]: make it private

File: app/src/main/java/io/neurolab/output/visual/FocusOMeterGLRenderer.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    static Random random = new Random();
```
Comment (adityastic) [inline app/src/main/java/io/neurolab/output/visual/FocusOMeterGLRenderer.java]: same

**Review by adityastic — CHANGES_REQUESTED** [review]

**Comment by adityastic** [comment]
@suresh-gandhi add this to your app level build.gradle file inside defaultConfig
` multiDexEnabled true
`

**Comment by suresh-gandhi** [comment]
@adityastic I'll do the changes. Thanks for the peer review.

**Comment by suresh-gandhi** [comment]
@iamareebjamal Could you please review.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**